<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartitionManagerKCMBase</class>
 <widget class="QWidget" name="PartitionManagerKCMBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>712</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="m_SplitterVertical">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="KToolBar" name="m_ToolBar" native="true"/>
     <widget class="QSplitter" name="m_SplitterHorizontal">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="ListOperations" name="m_ListOperations" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="PartitionManagerWidget" name="m_PartitionManagerWidget" native="true"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PartitionManagerWidget</class>
   <extends>QWidget</extends>
   <header>gui/partitionmanagerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListOperations</class>
   <extends>QWidget</extends>
   <header>gui/listoperations.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KToolBar</class>
   <extends>QWidget</extends>
   <header location="global">ktoolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
